spec/rails_helper.rb

```diff
@@ -96,6 +96,11 @@ class Analytics
     default_url_options = ApplicationController.default_url_options.merge(host: server_domain)
     self.default_url_options = default_url_options
     allow(Rails.application.routes).to receive(:default_url_options).and_return(default_url_options)
+    # temporary fix for local development feature tests
+    # remove when we get a new working version of Chromedriver
+    if ENV['CI'] != 'true'
+      Webdrivers::Chromedriver.required_version = '114.0.5735.90'
```

Comment:
I might expect to see this configuration someplace like [`support/capybara.rb`](https://github.com/18F/identity-idp/blob/main/spec/support/capybara.rb), but not a big deal if it's short-lived.

Reply:
Good idea. That works too. No need to reset it before each spec.